isso/css/isso.css

```diff
@@ -33,8 +33,6 @@
 
```

Comment:
This commit is titled `css: isso.css: Implement styling for .isso-is-page-author` but it does not add any styling to `.isso-is-page-author`.

Here's what makes most sense: Style the `.isso-page-author-suffix` class:
```css
.isso-comment > .isso-text-wrapper > .isso-comment-header .isso-page-author-suffix {
    font-weight: bold;
    color: #2c2c2c;
}
```
Bold font weight to match author class and color is author color `#555` slightly darkened.

Reply:
Oops, that happened because I squashed the commit where I removed the explicit color styling into that one and didn't rename it. I updated it now